src/components/Enquiry.js

```diff
@@ -57,8 +57,20 @@ const Enquiry = (props) => {
   const classes = useStyles();
 
   const handleKeyPress = (event) => {
-    if (event.charCode === 13 && event.target.value) {
-      setChfid(event.target.value);
+    const value = event.target.value;
+    const charCode = event.charCode;
+  
+    const isAlphaNumeric = /^[a-zA-Z0-9]*$/.test(value + String.fromCharCode(charCode));
+    const isLengthValid = value.length < 12 || charCode === 8;
```

Comment:
@dragos-dobre  / @dborowiecki  is there  a config for the max len ? @lruzicki  at least use a constant here

Reply:
I will put it in constants.js

Reply:
Constrait was added, I'll resolve the conversaiton and merge PR